pd/src/main.rs

```diff
@@ -48,9 +50,41 @@ enum Command {
         metrics_port: u16,
     },
 
-    /// Prints a sample `app_data` JSON object that can act as a template for
-    /// editing genesis configuration.
-    CreateGenesisTemplate,
+    /// Generates a directory structure containing necessary files to run a
+    /// testnet based on input configuration.
+    GenerateTestnet {
+        /// How many validator nodes to create configuration for.
+        #[structopt(short, long, default_value = "4")]
```

Comment:
Why is the default 4 here? I suspect most people who want to run a testnet node will probably only want to run one.

Reply:
Good question. Right now I'm optimizing the defaults to run a self-contained testnet (i.e. for the Penumbra team doing local development) that doesn't connect to any peers over the internet.

However you're right that most users wanting to operate a node connecting to the existing testnet will only want to run one.

These are two different use cases and I don't think it makes sense for users attempting to add an additional node to the testnet to use this script, they should instead have an alternative method documented that will connect to the correct seed peers for the running testnet.

Reply:
I'm not sure that there's actually a tension here -- users who want to run a node for the main testnet aren't generating their own testnet, they're joining an existing one. So I don't think they'd be using this command at all.